torch/nn/functional.py

```diff
@@ -2138,6 +2138,11 @@ def embedding(
 
     See :class:`torch.nn.Embedding` for more details.
 
+    .. warning::
+        To ensure correct gradients during backward, make sure the entries in the :attr:`weight`
+        matrix at the row specified by :attr:`padding_idx` are all zeroes. Alternatively, use
```

Comment:
Is that full rows or any arbitrary subset of the input?

Reply:
Full rows, the embedding matrix is 2D, and padding_idx gives us the row